cocos/ui/page-view.ts

```diff
@@ -703,7 +705,8 @@ export class PageView extends ScrollView {
     }
 
     // 是否超过自动滚动临界值
-    protected _isScrollable (offset: Vec3, index: number, nextIndex: number) {
+    // eslint-disable-next-line consistent-return
```

Comment:
Should not ignore this, just add another `return false` at the end of the function